### ISSUE TITLE
Search?

### ISSUE THREAD

**Issue by notriddle** [FSTDT/fstdt#11]
I'm probably beginning to sound like a broken record, but I think doing it the way GitHub Issues does it would be better than the Big Form From Hell that we currently use...

**Comment by spikedee** [comment]
Yeah. At the very least we'll collapse it into a combobox or something.
